Assets/Scripts/Layers/LayerTypes/LayerData.cs

```diff
@@ -11,16 +12,39 @@
 namespace Netherlands3D.Twin.Layers
 {
     [Serializable]
-    public class LayerData
+    public abstract class LayerData
     {
         [SerializeField, DataMember] protected Guid UUID = Guid.NewGuid();
         [SerializeField, DataMember] protected string name;
         [SerializeField, DataMember] protected bool activeSelf = true;
-        [SerializeField, DataMember] protected Color color = new Color(86f / 256f, 160f / 256f, 227f / 255f);
+        
+        /// <summary>
+        /// The default color of a layer.
+        /// 
+        /// This will influence how it is displayed in the layers side-panel, it does not automatically imply any
+        /// coloring in the styling of the layer but can be used to tell layers apart from one another in the layer
+        /// panel.
+        ///
+        /// Each type of layer could decide to use this value to influence the default styling by listening to the
+        /// ColorChanged event and applying the color to the relevant color field in the default Style, such as fill
+        /// for polygon vector layers, or stroke color for line polygon layers.
+        /// </summary>
+        [SerializeField, DataMember] protected Color color = new(86f / 256f, 160f / 256f, 227f / 255f);
```

Comment:
ik zou hier juist wel de default style (fill)color voor gebruiken, of is er een reden dat dat niet handig is?

Reply:
Ik vond het een lastige vraag; uberhaupt LayerData.Color is een beetje een lastige omdat iets niet maar 1 kleur heeft, en sommige dingen (zoals Obj's) helemaal geen kleur; of zelfs meerdere kleuren (stroke vs fill - bij ene polygon; wat is 'de' kleur)

Dus ik behandel het nu vergelijkbaar met Illustrator, waarbij de laagkleur niet een daadwerkelijk kleur van je illustratie voorstelt, maar een metadata kleur is die je o.a. kan gebruiken voor (selectie) outlines om duidelijk te maken op welke laag je aan het werk bent.